src/generator.ts

```diff
@@ -175,11 +224,25 @@ export class Generator {
     await createFiles(this.rootDir, files, true);
   }
 
+  private async _patchTsconfigJSON() {
+    const tsconfigFile = path.join(this.rootDir, 'tsconfig.json');
+    const files = new Map<string, string>();
+    if (!fs.existsSync(tsconfigFile)) {
+      files.set(`tsconfig.json`, this._readAsset(path.join('tsconfig.json')));
+    } else {
+      const tsconfigJSON = fs.readFileSync(path.join(this.rootDir, 'tsconfig.json'), 'utf-8');
+      const newJSON = tsconfigJSON.replace(/("include"[\s\S]*:[\s\S]\[[\s\S]*"src")/m, '$1, "playwright/playwright.d.ts"');
```

Comment:
Looks like this assumes they have `"src"` in their `include` already? Why is that a legit assumption?

Reply:
I'd say it is Okay for now...